lib/yard-sorbet/handlers/abstract_handler.rb

```diff
@@ -0,0 +1,23 @@
+# typed: strict
+# frozen_string_literal: true
+
+# Apllies an +@abstract+ tag to +abstract!+/+interface!+ modules (if not alerady present).
+class YARDSorbet::Handlers::AbstractHandler < YARD::Handlers::Ruby::Base
+  extend T::Sig
+
+  handles method_call(:abstract!), method_call(:interface!)
+  namespace_only
+
+  # The text accompanying the `@abstract` tag.
+  # @see https://github.com/lsegal/yard/blob/main/templates/default/docstring/html/abstract.erb
+  #   The `@abstract` tag template
+  TAG_TEXT = 'Descendants must implement the `abstract` methods below.'
```

Comment:
Can we also mention that class cannot be instantiated?

Reply:
Done! I'm doing this only when the `namespace` represents a class, so there's a tiny bit of extra logic.